src/core/Akka/Actor/RepointableActorRef.cs

```diff
@@ -366,7 +369,7 @@ private void SendMessage(object message, IActorRef sender)
                     }
                     else
                     {
-                        _messageQueue.Add(new Envelope { Message = message, Sender = sender });
+                        _messageQueue.AddLast(new Envelope { Message = message, Sender = sender });
```

Comment:
Append to the end of the linked list, consistent with what you'd expect from a queue structure.